src/app/contexts/EventContext/index.test.jsx

```diff
@@ -0,0 +1,161 @@
+import React, { useContext, useEffect, useState } from 'react';
```

Comment:
I feel this is mostly testing implementation details. I would have thought you only need to test that EventContext.Provider is called with expected value and children.

Reply:
The EventContext doesn't modify the DOM in a way that snapshot tests can detect, so we need to assert that import functions like `useWindowEvent` and `useHandlerMap` are invoked.

We also need to test `useClickTracker`, a function that only exists within `EventContextProvider` and that we can't test in isolation.